app/src/main/java/com/example/henzoshimada/feeltrip/Mood.java

```diff
@@ -67,7 +67,7 @@ public static int getSize() {
     7 ----> emoji image
     */
 
-    private static boolean[] stateVector;
+    private boolean[] stateVector;
```

Comment:
This is very problematic. static variables don't behave normally in file operations. I don't know why we need to declare them as static, but there are more static variables in mood class. They are most likely not gonna work in in offline sync when app gets restarted. I didn't change them since i don't know how they are used elsewhere.